bot.py

```diff
@@ -5,12 +5,11 @@
 import datetime
 import json
 import structlog
-import asyncio
 import async_timeout
-import aioredis
 import re
 from dotenv import load_dotenv
 from PIL import Image
+from redis import asyncio as aioredis
```

Comment:
I would just leave this package as named `redis` (remove `as aioredis` and replace all other `aioredis` instances with `redis`).

Reply:
Added new commit to do it